articles/anomaly-detection/index.md

```diff
@@ -16,82 +16,99 @@ useCase: customize-anomaly-detection
 
 Auth0 provides built-in tools to detect anomalies and stop malicious attempts to access your application. Anomaly detection can alert you and your users of suspicious activity, as well as block further login attempts. You can set your preferences on the notifications that get sent and you can decide whether to block a suspicious IP address or not.
 
-## What Anomaly Detection Provides
+Currently Auth0 has two types of **shields** you can enable to handle anomalies and attacks.  
 
-Currently Auth0 has three types of **shields** you can enable to handle anomalies and attacks.  A **shield** specifies the **action** you wish to take given a specific **trigger**.
+* Brute-Force Protection
+* Breached Password Detection
 
-A **trigger** is a suspicious event that is detected when someone is trying to login to your system, or there may have been a breached password with another 3rd party service.
+A **shield** specifies the **action** you wish to take given a specific **trigger**. A **trigger** is a suspicious event that is detected when someone is trying to login to your system, or there may have been a breached password with another third party service.
 
-## Shields
-
-### Brute-Force Protection
+## Brute-Force Protection
 
 There are two different triggers for the brute-force protection shield, for two slightly different attack scenarios.
 
-**Trigger:** *10* failed login attempts into a single account from the same IP address.
+* 10 failed login attempts from the same IP address
```

Comment:
I think we need to mention 'for the same username and same IP address'. We might want to also say 'consecutive login attempts', since any successful login from the IP address for that username will clear this counter.

Reply:
Changed to: * 10 consecutive failed login attempts for the same user and from the same IP address